src/runtime_src/driver/zynq/drm/zocl/sched_exec.c

```diff
@@ -1246,8 +1300,20 @@ static int
 add_cmd(struct sched_cmd *cmd)
 {
 	int ret = 0;
+	int num_masks = 0;
+	int mask_idx = 0;
 
 	SCHED_DEBUG("-> add_cmd\n");
+
+	/*
+	 * We always check if given cmd is set wrong CU mask.
+	 */
+	num_masks = cu_masks(cmd);
+	for (mask_idx = 0; mask_idx < num_masks; mask_idx++) {
+		if (cmd->packet->data[mask_idx] & cmd->exec->cu_valid[mask_idx])
```

Comment:
Only check this for start CU cmd.

if  (opcode(cmd) == ERT_START_CU) {
...
if (cmd->packet->data[mask_idx] & cmd->exec->cu_valid[mask_idx])
...
}